src/arkit.cs

```diff
@@ -1045,6 +1158,128 @@ interface ARDirectionalLightEstimate {
 		[Export ("primaryLightIntensity")]
 		nfloat PrimaryLightIntensity { get; }
 	}
+
+	[iOS (12,0)]
+	[NoWatch, NoTV, NoMac]
+	[BaseType (typeof(ARConfiguration))]
+	interface ARImageTrackingConfiguration {
+		[Export ("autoFocusEnabled")]
+		bool AutoFocusEnabled { [Bind ("isAutoFocusEnabled")] get; set; }
+
+		[Export ("trackingImages", ArgumentSemantic.Copy)]
+		NSSet<ARReferenceImage> TrackingImages { get; set; }
+
+		[Export ("maximumNumberOfTrackedImages")]
+		nint MaximumNumberOfTrackedImages { get; set; }
+	}
+
+	[iOS (12,0)]
+	[NoWatch, NoTV, NoMac]
+	[BaseType (typeof(ARConfiguration))]
+	interface ARObjectScanningConfiguration {
+		[Export ("autoFocusEnabled")]
+		bool AutoFocusEnabled { [Bind ("isAutoFocusEnabled")] get; set; }
+
+		[Export ("planeDetection", ArgumentSemantic.Assign)]
+		ARPlaneDetection PlaneDetection { get; set; }
+	}
+
+	[iOS (12,0)]
+	[NoWatch, NoTV, NoMac]
+	[BaseType (typeof(ARAnchor))]
```

Comment:
Considering the `init*` don't accept `nil` is seems unlikely the default constructor should be allowed.